co-log-core/src/Colog/Core/Class.hs

```diff
@@ -18,11 +17,13 @@ TODO: laws
 class HasLog env msg m where
     getLogAction :: env -> LogAction m msg
     setLogAction :: LogAction m msg -> env -> env
+    setLogAction = overLogAction . const
+    {-# INLINE setLogAction #-}
+    overLogAction :: (LogAction m msg -> LogAction m msg) -> env -> env
+    overLogAction f env = setLogAction (f $ getLogAction env) env
+    {-# INLINE overLogAction #-}
+    {-# MINIMAL getLogAction, (setLogAction | overLogAction) #-}
```

Comment:
Can you, please, run haddock command for `co-log-core` and post screenshot of how it looks like? I hope that syntax is correct and there should be no problems with rendering it... Also, it's usually a good practice to put `{-# MINIMAL #-}` pragma before all methods inside typeclass definition.